TR.html

```diff
@@ -141,7 +141,7 @@ <h2 id="status">Status of this Document</h2>
             This document is intended to become a <abbr title="World Wide Web Consortium">W3C</abbr> Recommendation.
           
             If you wish to make comments regarding this document, please send them to 
-            <a href="mailto:public-webapps@w3.org">public-webapps@w3.org</a> 
+             <a href="https://github.com/w3c/FileAPI/issues" target="_blank">https://github.com/w3c/FileAPI/issues</a>
```

Comment:
Also, ideally we put this information clearly in the heading. See e.g. https://url.spec.whatwg.org/.